skins/oasis/modules/templates/PageHeader_Index.php

```diff
@@ -1,145 +1,76 @@
+		<div class="header-column header-left">
+			<h1><?= $title ?></h1>
+			<?php if ( !empty( $button['action'] ) ) {
+				echo $app->renderView(
+					'MenuButton',
+					'Index',
+					$button
+				);
 			}
 
 			echo $curatedContentToolButton;
 
-			// TODO: use PageHeaderIndexExtraButtons hook for these buttons
-
-			// "Add a photo" button
-			if ( !empty( $isSpecialImages ) && !empty( $wg->EnableUploads ) ) {
-				echo Wikia::specialPageLink( 'Upload', 'oasis-add-photo', 'wikia-button upphotos', 'blank.gif', 'oasis-add-photo-to-wiki', 'sprite photo' );
-			}
-
-			// "Add a video" button
-			if ( !empty( $isSpecialVideos ) && !empty( $wg->EnableUploads ) && $showAddVideoBtn ): ?>
-				<a class="button addVideo" href="#" rel="tooltip"
-				   title="<?= wfMsg( 'related-videos-tooltip-add' ); ?>"><img src="<?= wfBlankImgUrl(); ?>"
-																			  class="sprite addRelatedVideo"/> <?= wfMsg( 'videos-add-video' ) ?>
-				</a>
-			<? endif;
-
 			// comments & like button
-			if (!$isWallEnabled) {
-				echo F::app()->renderView('CommentsLikes', 'Index', array('comments' => $comments));
-			}
-			foreach ($extraButtons as $button) {
-				echo $button;
+			if ( !$isWallEnabled ) {
+				echo $app->renderView(
+					'CommentsLikes',
+					'Index',
+					[ 'comments' => $comments ]
+				);
 			}
 
-			// "pages on this wiki" counter
-			if (!is_null($tallyMsg)) {
-				?>
-				<div class="tally">
-					<?= $tallyMsg ?>
-				</div>
-			<?php
+			foreach ( $extraButtons as $button ) {
+				echo $button;
 			}
 
-			// render page type line
-			if ( !empty( $pageSubtitle ) ) {
-				?>
-				<h2><?= $pageSubtitle ?></h2>
-			<?php
-			}
+			if ( !empty( $pageSubtitle ) ) { ?>
+				<h2 class="breadcrumbs"><?= $pageSubtitle ?></h2>
+			<?php }
 
-			// MW subtitle
-			// include undelete message (BugId:1137)
-			if ( !empty( $subtitle ) ) {
-				?>
+			if ( !empty( $subtitle ) ) { ?>
 				<div class="subtitle"><?= $subtitle ?></div>
+			<?php } ?>
+		</div>
+		<div class="header-column header-right">
 			<?php
-			}
-			?>
-		</header>
-	<? else: ?>
-		<header id="WikiaPageHeader" class="WikiaPageHeader wikia-page-header">
-			<div class="header-container">
-				<div class="header-column header-title">
-					<h1><?= !empty( $displaytitle ) ? $title : htmlspecialchars( $title ) ?></h1>
-					<?php if ( !empty( $pageSubtitle ) ): ?>
-						<h2><?= $pageSubtitle ?></h2>
-					<? endif;
-					if ( !empty( $subtitle ) ): ?>
-						<div class="subtitle"><?= $subtitle ?></div>
-					<? endif; ?>
+			// don't show share buttons on Special:Videos as we want the tally instead
+			if ( !empty( $wg->EnablePageShareExt && !$isSpecialVideos ) ) { ?>
+				<div id="pageShareContainer" class="page-share-container">
+					<?= $app->renderView( 'PageShare', 'Index' ); ?>
 				</div>
-				<div class="header-column header-tally">
-					<? if ( !empty( $wg->EnablePageShareExt ) ): ?>
-						<div id="PageShareContainer" class="page-share-container">
```

Comment:
Does the [PageShare JS](https://github.com/Wikia/app/blob/dev/extensions/wikia/PageShare/scripts/PageShare.js#L40) need updating as well due to the case change on the ID?